File: problems/euler_001.js
function sum3or5(n) {
  var sum = 0;
  for (var i = 0; i < n; i ++) {
    if ((i%3 == 0) || (i%5 == 0)) {
      sum += i
    }
  }

  return sum
}

console.log(sum3or5(1000));

File: problems/_001/euler_001.c
#include <stdio.h>

int main(void)
{
    int sum = 0;
    int i;

    for (i = 0; i < 10; i++) {
        if ((i % 3 == 0) || (i % 5 == 0))
            sum += i;
    }

    printf("%d\n", sum);
    return sum;
}
